cachix-api/src/Cachix/Api.hs

```diff
@@ -86,6 +86,12 @@ data BinaryCacheAPI route
             :> Summary "Given a list of store hashes, return a list of those that are missing"
             :> ReqBody '[JSON] [Text]
             :> Post '[JSON] [Text],
+        narURL
+          :: route
+               :- CachixAuth
+               :> "narurl"
+               :> Capture "nar" NarFileName
+               :> Get '[JSON] Text,
```

Comment:
The `Text` here is probably a URL. Why is it special or when does a cachix client need to use this endpoint?

Reply:
Client doesn't use this endpoint.